plugins/events-backend-module-bitbucket-server/api-report.md

```diff
@@ -0,0 +1,18 @@
+## API Report File for "@backstage/plugin-events-backend-module-bitbucket-server"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+import { EventParams } from '@backstage/plugin-events-node';
+import { EventsService } from '@backstage/plugin-events-node';
+import { SubTopicEventRouter } from '@backstage/plugin-events-node';
+
+// @public
+export class BitbucketServerEventRouter extends SubTopicEventRouter {
```

Comment:
Skip exporting this, just export the module as the `default` export instead

Reply:
@Rugvip The module requires this class, wouldn't I need to keep this exported then for the module/service to use? Or am I confusing terminology?